Description: normalize "linux/arm64" to "linux/arm64/v8" instead of the other direction
Forwarded: no; presumably, the direction this went was intentional (Tianon just disagrees, and the normalization allows him that)

diff --git a/platforms/database.go b/platforms/database.go
index 2e26fd3b4..5ef7d74d1 100644
--- a/platforms/database.go
+++ b/platforms/database.go
@@ -87,8 +87,8 @@ func normalizeArch(arch, variant string) (string, string) {
 	case "aarch64", "arm64":
 		arch = "arm64"
 		switch variant {
-		case "8", "v8":
-			variant = ""
+		case "", "8":
+			variant = "v8"
 		}
 	case "armhf":
 		arch = "arm"
